app/views/admin_mailer/member_unpaid_over_x_months.html.haml

```diff
@@ -0,0 +1,26 @@
+- provide :signoff do
+  = render 'admin_mailer/signoff'
+
+- message_scope = 'mailers.admin_mailer.member_unpaid_over_x_months.message_text'
+
+= render layout: 'layouts/mail_to_admin' do
+
+  %p= t('these_are_unpaid',  scope: message_scope, num_months: @num_months)
+
+  - inherit_border_padleft = "border: inherit; padding-left: inherit;"
```

Comment:
I'm sure there's a reason, but curious as to why a class couldn't be used?

Reply:
Because dealing with mail formatting is a PITA.
You can't really use CSS in email for this (at least not with how these emails are being sent).
So everything has to be "hand coded" (no CSS goodness).

Reply:
very true. 
Divs usually wont work either, has to be tables.